spec/rails_helper.rb

```diff
@@ -50,7 +50,7 @@
 
 Capybara.javascript_driver = :selenium_chrome_headless
 
-if ENV['NOT_HEADLESS']
+if EnvUtilities.load_boolean(name: 'HEADLESS', default: false)
```

Comment:
This changes the CLI var that spins up the feature specs in headless mode (or not).